package/butterfly_angular/analysis_options.yaml

```diff
@@ -0,0 +1,16 @@
+analyzer:
```

Comment:
Ooh! Can this file be at the root of the repo?

Reply:
Yes, Done